dtls.c

```diff
@@ -4015,6 +4015,10 @@ handle_handshake(dtls_context_t *ctx, dtls_peer_t *peer, uint8 *data, size_t dat
           netq_remove(&peer->handshake_params->reorder_queue, node);
           next = 1;
           res = handle_handshake_msg(ctx, peer, node->data, node->length);
+
+          // free message data
```

Comment:
I know, it's pendantic: But could you please use the `/* ... */` comment form even for single-line comments?

Reply:
Sure :-)

Reply:
Done.